metadata-db-app/src/cmr/metadata_db/migrations/077_remove_email_column.clj

```diff
@@ -0,0 +1,16 @@
+(ns cmr.metadata-db.migrations.077-remove-email-column
```

Comment:
this ns/filename should contain the word `subscription` somewhere

Reply:
done